src/test/java/com/rackspace/ceres/app/web/QueryControllerTest.java

```diff
@@ -52,11 +52,11 @@ public void testQueryApi() {
 
     Flux<QueryResult> result = webTestClient.get()
         .uri(uriBuilder -> uriBuilder.path("/api/query")
-            .queryParam("tenant", "t-1")
             .queryParam("metricName", "cpu-idle")
             .queryParam("tag", "os=linux")
             .queryParam("start", "1d-ago")
             .build())
+        .header("X-Tenant-Id", "t-1")
```

Comment:
Also here please duplicate the test case and have one still use the tenant query parameter.

Reply:
Do we need to duplicate here as the next test case `testQueryApiWithAggregator` is the one with `tenantParam` only?

Reply:
Sorry I didn't expand the diff to check that. One test case out of the suite seems fine.